packages/sui-helpers/cli.js

```diff
@@ -106,7 +107,20 @@ function getSpawnProcess (bin, args, options = {}) {
     { shell: true, stdio: 'inherit', cwd: process.cwd() },
     options
   )
-  return processSpawn(bin, args, options)
+  return processSpawn(...getArrangedCommand(bin, args, options))
+}
+
+/**
+ * Returns modified command to work on linux, osx and windows
+ * @param  {String} bin     Binary path or alias
+ * @param  {Array} args    Array of args, like ['npm', ['run', 'test']]
+ * @param  {Object} opts to pass to child_process.spawn call
+ * @returns {Object} {bin, args, options}
+ */
+function getArrangedCommand (bin, args, opts) {
+  return path.isAbsolute(bin) // check if it's a file or an alias
```

Comment:
I think you could explain a bit better with code/comments why file or alias is checked with `isAbsolute` and why that's enough to know that it is being run on a windows/unix env

Reply:
Done !